app/models/force/application_member.rb

```diff
@@ -32,8 +32,24 @@ class ApplicationMember < Force::ObjectBase
       { custom_api: '', domain: 'residence_address', salesforce: 'Residence_Address' },
       { custom_api: 'state', domain: 'state', salesforce: 'State' },
       { custom_api: 'zip', domain: 'zip_code', salesforce: 'Zip_Code' },
+      # Demographics, just for primary applicants
+      { custom_api: 'ethnicity', domain: 'ethnicity', salesforce: '?'},
+      { custom_api: 'gender', domain: 'gender', salesforce: '?'},
+      { custom_api: 'sexualOrientation', domain: 'sexual_orientation', salesforce: '?'},
+      { custom_api: 'race', domain: 'race', salesforce: '?'},
+      { custom_api: 'genderOther', domain: 'gender_other', salesforce: '?'},
+      { custom_api: 'sexualOrientationOther', domain: 'sexual_orientation_other', salesforce: '?'}
     ].freeze
 
+    def to_custom_api
+      custom_api_fields = super
+      # Convert array date into string
+      if @fields.domain and @fields.domain['date_of_birth']
```

Comment:
I think we could do something fancy with converting dates using the "object" key in the field definitions, but that also seems out of scope for this PR